From: Owen Synge <osynge@suse.com>
Date: Mon, 11 Aug 2014 11:26:04 +0200
Subject: [PATCH] MultiPatch taken from trunk

THis patxh shodul be a series of cherry picks but it wall went wrong.
Do this patch again from original patches.
---
 src/init-ceph.in | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/src/init-ceph.in b/src/init-ceph.in
index ee1dc41..80476ce 100644
--- a/src/init-ceph.in
+++ b/src/init-ceph.in
@@ -4,7 +4,7 @@
 
 ### BEGIN INIT INFO
 # Provides:          ceph
-# Default-Start:     3 5
+# Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
 # Required-Start:    $remote_fs $named $network $time
 # Required-Stop:     $remote_fs $named $network $time
@@ -148,20 +148,12 @@ case $1 in
     --btrfs | --fsmount)
 	    dofsmount=1
 	    ;;
-    --xfs)
-	    doxfs=1
-	    dobtrfs=0
-	    ;;
     --nobtrfs | --nofsmount)
 	    dofsmount=0
 	    ;;
     --btrfsumount | --fsumount)
 	    dofsumount=1
 	    ;;
-    --xfsumount)
-	    doxfsumount=1
-	    dobtrfsumount=0
-	    ;;
     --conf | -c)
 	    [ -z "$2" ] && usage_exit
 	    options="$options $1"
